Link from README to the release script (#6458)
python/typeshed: README.md

    $ pip install types-six types-requests

These PyPI packages follow [PEP 561](http://www.python.org/dev/peps/pep-0561/)

> and are automatically released (daily) by

Reviewer: If I understood correctly, the release always happens within 3 hours, but it would be good if another typeshed maintainer confirms this.

Reviewer 2: I'm pretty sure 3 hours is right. Elsewhere you linked to the crontab and the cron runs every 3 hours.

Author: Oh you right. I read the cron too fast and saw every day at 3, where it's "every hours divisible by 3" so 8 times a day

Author: You can check how often the CI runs: https://github.com/typeshed-internal/stub_uploader/actions/workflows/update_stubs.yml

I was put off by the name of the workflow: "Daily update of stubs from typeshed to PyPI".

Author: Ok, fixed :)

[typeshed internal machinery](https://github.com/typeshed-internal/stub_uploader).

> Also starting from version 0.900 mypy will provide an option to automatically install

Reviewer: 0.900 has been released for a while now.

```suggestion
Also, starting from version 0.900, you can use mypy's `--install-types` option to automatically install
```

    $ pip install types-six types-requests

These PyPI packages follow [PEP 561](http://www.python.org/dev/peps/pep-0561/)
and are automatically released (multiple times a day, when needed) by
[typeshed internal machinery](https://github.com/typeshed-internal/stub_uploader).

PyCharm, pytype etc. work in a similar way, for more details see documentation

> for the type-checking tool you are using.

Reviewer 2: Now this paragraph doesn't make sense, because it meant "similar to mypy". I would rewrite it as "Type checkers should be able to use these stub packages when installed. For more details, see the documentation for your type checker."
